src/components/profile/ConsultantInformation.tsx

```diff
@@ -53,9 +55,18 @@ export const ConsultantInformation = () => {
 			.then(() => {
 				reloadUserData().catch(console.log);
 				setInitialDisplayName(editedDisplayName);
-				setIsEditEnabled(false);
```

Comment:
Das sollte hier bleiben. Im catch fall kann man ja nochmal auf speichern klicken und es nochmal versuchen.
Wenn wir ihn zurück in den nicht editiermodus schicken dann sieht das weired aus weil er die geänderten Daten im nicht editiermodus sieht und dann hat man keine Ahnung ob jetzt gespeichert wurde oder nicht.

Reply:
Das hat ja nichts geändert. Aktuell wurde der Call im then und catch ausgeführt. Das ist dasselbe wie ein Aufruf in finally. Daher ändert sich hier nichts. Jetzt wird nur zusätzlich die Notification angezeigt

Reply:
Ahh sorry hatte ich nicht gesehen. Ich würde es trotzdem kicken. Wenn nix gespeichert wurde macht es keinen Sinn den edit modus zu verlassen. Das führt nur zu Verwirrung

Reply:
Okay lassen wir erstmasl so